src/main/java/com/hivemq/cli/commands/options/TlsOptions.java

```diff
@@ -138,85 +137,298 @@ private boolean useBuiltSslConfig() {
                 .build();
     }
 
-    private void setDefaultOptions() {
+    private @Nullable KeyManagerFactory buildKeyManagerFactory() throws Exception {
+        final KeyManagerFactory keyManagerFactory;
+
         final DefaultCLIProperties defaultCLIProperties =
                 Objects.requireNonNull(MqttCLIMain.MQTTCLI).defaultCLIProperties();
-
-        if (clientCertificateChain == null) {
+        final Path defaultKeystore = defaultCLIProperties.getKeystore();
+        final Path defaultClientCertificateChain = defaultCLIProperties.getClientCertificateChain();
+        final Path defaultClientPrivateKey = defaultCLIProperties.getClientPrivateKey();
```

Comment:
This also needs some validation. I would not allow the user to define contradictory default options.
I would only allow to define either the keystore or the certs directly.